ISSUES.md

```diff
@@ -207,6 +207,8 @@ You are not required to read the following guidelines before opening an issue. H
 
    Do not dispair if you can't figure it out from the begining, just share what you can and perhaps someone else will be able to help you at the forums.
 
+   If your setup involves any custom datasets, the best way to help us reproduce the problem is to create a [Google Colab notebook](https://colab.research.google.com/) that demonstrates the issue and once you verify that the issue still exists, include a link to that notebook in the Issue. Just make sure that you don't copy-n-paste the location bar url of the open notebook - as this is private and we won't be able to open it. Instead, you need to click on `Share` in the right upper corner of the notebook, select `Get Link` and then copy-n-paste the public link it will give to you.
```

Comment:
```suggestion
   If your setup involves any custom datasets, the best way to help us reproduce the problem is to create a [Google Colab notebook](https://colab.research.google.com/) that demonstrates the issue and once you verify that the issue still exists, include a link to that notebook in the Issue. Just make sure that you don't copy and paste the location bar url of the open notebook - as this is private and we won't be able to open it. Instead, you need to click on `Share` in the right upper corner of the notebook, select `Get Link` and then copy and paste the public link it will give to you.
```

Reply:
I did it again! Too much rock-n-roll! Thank you, @sgugger!